requirements.txt

```
@@ -1,13 +1,13 @@
 argcomplete==1.12.2
-coverage==5.3
-importlib_metadata==2.0.0
+coverage==5.5
+importlib_metadata==3.10.0
```

`importlib-metadata` is only needed with older Python versions:

```suggestion
importlib_metadata==3.10.0; python_version < '3.8'
```